docs/adminguide/frozen_users.rst

```diff
@@ -0,0 +1,114 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+.. _doc_adminguide_frozen_users:
+
+
+Frozen users
+============
+
+It is possible for server administrators to freeze specific Parsec users.
+
+This feature is not part of the Parsec revocation system, but a way for system administrators to temporarily block users from connecting to the server while waiting for organization administrators to actually revoke them. It is exposed as an HTTP route that requires the administration token. This way, it is possible for existing directory services to automatically freeze users that have been removed from the directory.
```

Comment:
I would rephrase it a little bit:
* add small note of what revocation system is about.
* avoid using "feature" as it is a more spec-related term.
* emphasize main goal of the freeze/unfreeze mechanism 
```suggestion
Frozen users will be temporarily blocked from connecting to the Parsec server while waiting for an organization administrator to actually revoke them. This operation is thus not part of the Parsec revocation system (i.e. the definitive removal of the user's rights).

The freeze mechanism enables to automatically block users who have been deleted from a directory service (such as OpenLDAP or AD), as it is exposed in the form of HTTP routes that only require an administration token. 
```

Reply:
[70d6ef2](https://github.com/Scille/parsec-cloud/pull/5842/commits/70d6ef29491522ed83c28a1dcd9f084b78896ef2)